client/components/banner/index.jsx

```diff
@@ -65,14 +66,39 @@ class Banner extends Component {
 	};
 
 	getHref() {
-		const { href, feature, siteSlug } = this.props;
+		const { feature, plan, siteSlug } = this.props;
 
-		if ( ! href && siteSlug ) {
-			if ( feature ) {
-				return `/plans/${ siteSlug }?feature=${ feature }`;
-			}
-			return `/plans/${ siteSlug }`;
+		const baseUrl = `/plans/${ siteSlug }`;
+		let selectedPlan = 'Personal';
+		switch ( plan ) {
```

Comment:
All of this logic would be unnecessary if you directly pass `plan` as a query arg.

Reply:
If I use plan slug from `planType` instead of `planName` this indeed can go :)